pkg/sfu/streamallocator/track.go

```diff
@@ -134,6 +134,10 @@ func (t *Track) WritePaddingRTP(bytesToSend int) int {
 	return t.downTrack.WritePaddingRTP(bytesToSend, false, false)
 }
```

Comment:
Can potentially remove this method now as we move to using RTX for probing, but leaving it for some more testing. When using this, seeing occasional PLIs. Spent way too much time trying to chase as to why, but came up empty. But, may try it again some time to debug.